src/app/shell/personal-cabinet/provider/create-workshop/create-description-form/create-description-form.component.html

```diff
@@ -1,43 +1,65 @@
 <form [formGroup]="DescriptionFormGroup" fxLayout="column" fxLayoutAlign="center space-between" class="step">
-  <app-image-form-control *ngIf="isImagesFeature" [imgMaxAmount]="10" [cropperConfig]="cropperConfig" [label]="'Фото'"
-    formControlName="imageFiles" [imageIdsFormControl]="DescriptionFormGroup.get('imageIds')">
+  <app-image-form-control
+    *ngIf="isImagesFeature"
+    [imgMaxAmount]="10"
+    [cropperConfig]="cropperConfig"
+    [label]="'Фото'"
+    formControlName="imageFiles"
+    [imageIdsFormControl]="DescriptionFormGroup.get('imageIds')">
   </app-image-form-control>
 
-  <app-info-form *ngFor="let form of SectionItemsFormArray.controls" [infoEditFormGroup]="form"
-    [index]="SectionItemsFormArray.controls.indexOf(form)" [formAmount]="SectionItemsFormArray.controls.length"
-    [maxDescriptionLength]="validationConstants.MAX_DESCRIPTION_LENGTH_500" (deleteForm)="onDeleteForm($event)">
+  <app-info-form
+    *ngFor="let form of SectionItemsFormArray.controls"
+    [infoEditFormGroup]="form"
+    [index]="SectionItemsFormArray.controls.indexOf(form)"
+    [formAmount]="SectionItemsFormArray.controls.length"
+    [maxDescriptionLength]="validationConstants.MAX_DESCRIPTION_LENGTH_500"
+    (deleteForm)="onDeleteForm($event)">
   </app-info-form>
 
-  <button *ngIf="SectionItemsFormArray.controls.length < 10" [disabled]="SectionItemsFormArray.invalid" class="btn-add"
-    mat-button (click)="onAddForm()">
+  <button
+    *ngIf="SectionItemsFormArray.controls.length < 10"
+    [disabled]="SectionItemsFormArray.invalid"
+    class="btn-add"
+    mat-button
+    (click)="onAddForm()">
     <mat-icon class="mat-icon-add">add</mat-icon>{{ 'BUTTONS.ADD_ANOTHER_SECTION' | translate }}
   </button>
 
   <label class="step-label"> {{ 'FORMS.LABELS.SUPPORT_PERSONS_WITH_DISABILITIES' | translate }} </label>
-  <mat-radio-group [formControl]="disabilityOptionRadioBtn" color="primary" fxLayout="column"
-    fxLayoutAlign="center space-between">
+  <mat-radio-group [formControl]="disabilityOptionRadioBtn" color="primary" fxLayout="column" fxLayoutAlign="center space-between">
     <mat-radio-button #radio1 name="radioBtn1" [value]="false">{{ 'NOT_AVAILABLE' | translate }}</mat-radio-button>
     <mat-radio-button #radio2 name="radioBtn2" [value]="true">{{ 'AVAILABLE' | translate }}</mat-radio-button>
-    <input matInput class="step-input" [ngClass]="{ 'disabled-field': !disabilityOptionRadioBtn.value }"
-      placeholder="{{ 'FORMS.PLACEHOLDERS.SPECIAL_EQUIPMENT' | translate }}" formControlName="disabilityOptionsDesc"
-      autocomplete="off" appTrimValue />
-    <app-validation-hint [validationFormControl]="DescriptionFormGroup.get('disabilityOptionsDesc')"
-      [minCharacters]="validationConstants.INPUT_LENGTH_1" [maxCharacters]="validationConstants.INPUT_LENGTH_256">
+    <input
+      matInput
+      class="step-input"
+      [ngClass]="{ 'disabled-field': !disabilityOptionRadioBtn.value }"
```

Comment:
When we have only one class, use [class.class-name]="condition" intead of ngClass